common/src/python/identifiers/identifiers_repository.py

```diff
@@ -0,0 +1,119 @@
+"""Repository for Identifiers.
+
+Inspired by
+https://github.com/cosmicpython/code/tree/chapter_02_repository_exercise
+"""
+
+from typing import List, Optional, overload
+
+from identifiers.identifiers_tables import metadata
+from identifiers.model import Identifier
+from inputs.parameter_store import RDSParameters
+from sqlalchemy import create_engine
+from sqlalchemy.exc import NoResultFound
+from sqlalchemy.orm import Session, sessionmaker
+
+
+class IdentifierRepository:
+    """Repository for Identifier records.
+
+    Assumes the Identifier class is mapped to the identifier table.
+    """
+
+    def __init__(self, session: Session) -> None:
+        self.__session = session
```

Comment:
I think the session need to be closed somewhere.

Reply:
Good point. It seems like close() is more of a reset, but the recommendation is to use it anyway. 
I'll look at switching to using a resource block which makes sure close is called

Reply:
I moved the "create_from" method out of the repository class and changed it to create a session. 

This can then be used in a resource block

```
identifiers_session = create_session(parameters)
with identifier_session as session:
    repository = IdentifierRespository(session)
```

which ensures that close is done at the end of the block.